azurefunctions-extensions-bindings-blob/azurefunctions/extensions/bindings/blob/storageStreamDownloader.py

```diff
@@ -42,3 +41,16 @@ def get_sdk_type(self):
             return blob_client.download_blob()
         else:
             return None
+
```

Comment:
can we extract this function to a util class or baseclass?

Reply:
Moved to a utils class